runner/run_benchmark.py

```diff
@@ -25,61 +25,60 @@
+    gather_persistence_files
+from utils.llm import get_model, GPT_4_TURBO_BEST
 from utils.constants import MAIN_DIR
 
 
 def get_chat_session(name: str, max_prompt_size: Optional[int], run_name: str) -> ChatSession:
     kwargs = {"max_prompt_size": max_prompt_size} if max_prompt_size is not None else {}
     kwargs["run_name"] = run_name
 
-    if name in ["gpt", "gpt-4"]:
-        return GPTChatSession(model="gpt-4", **kwargs)
-    elif name == "gpt-3.5-turbo":
-        return GPTChatSession(model="gpt-3.5-turbo", **kwargs)
-    elif name in ["gpt-4-1106", "gpt-4-0125"]:
-        return GPTChatSession(model=f"{name}-preview", **kwargs)
-    elif name == "ts-gpt-3.5-turbo":
-        return TimestampGPTChatSession(model="gpt-3.5-turbo", **kwargs)
-    elif name in ["ts-gpt-4-1106", "ts-gpt-4-0125"]:
-        model = name.removesuffix("ts-") + "-preview"
-        return TimestampGPTChatSession(model=model, **kwargs)
-    elif name == "memgpt":
+    if name == "memgpt":
         return MemGPTChatSession(run_name=run_name)
-
-    elif name == "langchain_sb_a":
-        return LangchainAgent(model_name="gpt-3.5-turbo-instruct", mem_type=LangchainMemType.SUMMARY_BUFFER, **kwargs)
-    elif name == "langchain_kg_a":
-        return LangchainAgent(model_name="gpt-3.5-turbo-instruct", mem_type=LangchainMemType.KG, **kwargs)
-    elif name == "langchain_ce_a":
-        return LangchainAgent(
-            model_name="gpt-3.5-turbo-instruct", mem_type=LangchainMemType.CONVERSATION_ENTITY, **kwargs
-        )
-    elif name == "ltm_agent_1":
-        return LTMAgentWrapper(model="gpt-4-0125-preview",
-                               variant=LTMAgentVariant.QG_JSON_USER_INFO, **kwargs)
-    elif name == "ltm_agent_2":
-        return LTMAgentWrapper(model="gpt-4-0125-preview",
-                               variant=LTMAgentVariant.SEMANTIC_ONLY, **kwargs)
-    elif name == "ltm_agent_3":
-        return LTMAgentWrapper(model="gpt-4-0125-preview",
-                               variant=LTMAgentVariant.TEXT_SCRATCHPAD, **kwargs)
-    elif name == "length_bias":
-        return LengthBiasAgent(model="gpt-4-0125-preview", **kwargs)
-    elif name.startswith("cost("):
+    if name.startswith("langchain_"):
+        suffix = "_".join(name.split("_")[1:])
+        mem_type = dict(
+            sb_a=LangchainMemType.SUMMARY_BUFFER,
+            kg_a=LangchainMemType.KG,
+            ce_a=LangchainMemType.CONVERSATION_ENTITY,
+        ).get(suffix, None)
+        if mem_type is None:
+            raise ValueError(f"Unrecognized LangChain memory type {repr(suffix)}.")
+        return LangchainAgent(model_name="gpt-3.5-turbo-instruct", mem_type=mem_type, **kwargs)
+    if name.startswith("ltm_agent_"):
+        agent_nr = "_".join(name.split("_")[2:])
+        variant = {
+            "1": LTMAgentVariant.QG_JSON_USER_INFO,
+            "2": LTMAgentVariant.SEMANTIC_ONLY,
+            "3": LTMAgentVariant.TEXT_SCRATCHPAD,
+        }.get(agent_nr, None)
```

Comment:
This form is inconsistent with the one above. Pick a style of creating a dict and use it universally.

Reply:
I do prefer using `dict` because it looks more pythonic, like when you call a function with keyword arguments, but it's not always possible. If the elements contain special characters or start with a digit, then I have no other option than using the string-based approach. I can stick to the dict literal in this case though, if you think it's for the best.

Reply:
Ah I see, okay then I think its okay. it was just confusing to see both styles so close to each other without knowing why.

Reply:
I agree in that it's confusing, being so close to each other.